RNZendeskChat.d.ts

```diff
@@ -68,7 +68,7 @@ declare module "react-native-zendesk-chat" {
 		 * Must be called before calling startChat/setVisitorInfo
 		 * - (Advanced users may configure this natively instead of calling this from JS)
 		 */
-		init: (zendeskAccountKey: string) => void;
+		init: (zendeskAccountKey: string, appId: string) => void;
```

Comment:
I guess this new parameter should have an `?` mark here, as it is optional.

Reply:
Exactly ;)

Reply:
```suggestion
		init: (zendeskAccountKey: string, appId?: string) => void;
```